tests/capi/doc/chfl_atom/property.c

```diff
@@ -4,22 +4,38 @@
 #include <chemfiles.h>
 #include <assert.h>
 #include <stdlib.h>
+#include <string.h>
 
 int main() {
     // [example]
+    // Set property
     CHFL_ATOM* atom = chfl_atom("Na");
     CHFL_PROPERTY* property = chfl_property_double(-23);
 
     chfl_atom_set_property(atom, "this", property);
     chfl_property_free(property);
 
+    // Get property
     property = chfl_atom_get_property(atom, "this");
 
     double value = 0;
     chfl_property_get_double(property, &value);
     assert(value == -23);
 
+    // Get all properties
```

Comment:
I would create a separated example for this function, as it is more complex than the two other ones